client/src/components/help/HelpModalContainer.tsx

```diff
@@ -0,0 +1,35 @@
+import useHelpModal from '../../hooks/useHelpModal';
+import HelpModalUI from './HelpModalUI';
+import { HelpModalContainerProps } from '@/types/help.types';
+
+const HelpModalContainer = ({ isModalOpened, handleCloseModal, handleKeyDown }: HelpModalContainerProps) => {
+  const {
+    pageData,
+    pageIndex,
+    setPageIndex,
+    pagenation,
+    dotLottieRefCallback,
+    handleTouchStart,
+    handleTouchEnd,
+    handleTouchMove,
+    changePageIndex,
+  } = useHelpModal(isModalOpened);
+  return (
+    <HelpModalUI
+      isModalOpened={isModalOpened}
+      handleCloseModal={handleCloseModal}
+      handleKeyDown={handleKeyDown}
+      pageData={pageData}
+      pageIndex={pageIndex}
+      setPageIndex={setPageIndex}
+      pagenation={pagenation}
+      dotLottieRefCallback={dotLottieRefCallback}
+      handleTouchStart={handleTouchStart}
+      handleTouchEnd={handleTouchEnd}
+      handleTouchMove={handleTouchMove}
+      changePageIndex={changePageIndex}
+    />
+  );
+};
```

Comment:
```suggestion
  const helpModalProps = useHelpModal(isModalOpened);
  return (
    <HelpModalUI {...helpModalProps} />
  );
};
```
`props`가 많아서 위와 같이 spread해서 쓸 수 있을 것 같아요!